src/main/scala/part5twitter/SentimentAnalysis.scala

```diff
@@ -25,7 +25,7 @@ object SentimentAnalysis {
     // split the text into sentences and attach scores to each
     val sentences = annotation.get(classOf[CoreAnnotations.SentencesAnnotation]).asScala
     val sentiments = sentences.map { sentence: CoreMap =>
-      val tree = sentence.get(classOf[SentimentCoreAnnotations.AnnotatedTree])
+      val tree = sentence.get(classOf[SentimentCoreAnnotations.SentimentAnnotatedTree])
       // convert the score to a double for each sentence
```

Comment:
AnnotatedTree is no longer available, so used SentimentAnnotatedTree.

Reply:
👍